watertap/unit_models/anaerobic_digestor.py

```diff
@@ -477,7 +485,7 @@ def build(self):
         )
         # The value is taken from Maravelias' data
         self.energy_electric_flow_vol_inlet = Param(
-            initialize=0.029,
+            initialize=59.99279,
```

Comment:
This number seems very high. I figured thermal input would be the primary energy requirement and that electric energy input would be minimal. Are we sure about this value and associated units?